### DISCUSSION TITLE
Using Next Image with Thumbnail plugin

### DISCUSSION THREAD

**Discussion by thefinnomenon** [igordanchenko/yet-another-react-lightbox#173] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

I am using Next Image for the album view as well as the slider but the thumbnails are loading the full sized images. Could you provide an example of using the Thumbnail plugin with the Next Image component please?

### Describe the solution you'd like

The thumbnails in the slider should use Next Image to load a smaller version of the original images.

**Comment by igordanchenko** [comment, marked as answer]
You can re-use the same `NextJsImage` component for the thumbnail, just need to specify it as a `render.thumbnail` render function.

```tsx
<Lightbox
    open={open}
    close={() => setOpen(false)}
    slides={slides}
    plugins={[Thumbnails]}
    render={{ slide: NextJsImage, thumbnail: NextJsImage }}
/>
```

**Comment by thefinnomenon** [comment]
Thanks, that worked perfectly! I think it would be worth adding this to the docs in the Next section.

**Reply by igordanchenko** [reply]
Feel free to submit a PR 🤝 

Here is the source of the Next.js docs page - https://github.com/igordanchenko/yet-another-react-lightbox/blob/main/docs/examples/nextjs.md

I'll also re-open this discussion to make it more visible and easier to find.
